kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java

```diff
@@ -38,6 +39,8 @@
 
 public class CloudConnectionConfigurationsUi extends Composite {
 
+    protected static final Logger logger = Logger.getLogger(CloudConnectionConfigurationsUi.class.getSimpleName());
```

Comment:
This should use `org.slf4j.LoggerFactory`

Reply:
I think in the get part the slf4j does not work. But this is a leftover that we can remove.

Reply:
You are right, i forgot we are on GWT.